### ISSUE TITLE
Daisy architecture file missing sample rate

### ISSUE THREAD

**Issue by madskjeldgaard** [grame-cncm/faust#682]
Hello

It seems (from a glance) that the daisy architecture file only sets the sample rate of faust DSP and not accordingly in the hardware.

https://github.com/grame-cncm/faust/blob/master-dev/architecture/daisy/ex_faust.cpp#L126

In Daisy I think this is done kind of like this, using the SampleRate enum:
```cpp
patch.SetAudioSampleRate(SaiHandle::Config::SampleRate::SAI_96KHZ);
```

**Comment by sletz** [comment]
Correct,` -sr` is an option of the `faust2daisy` script (see https://github.com/grame-cncm/faust/tree/master-dev/architecture/daisy) but I don't remember exactly why ((-;
